android/install.md

```diff
@@ -2,7 +2,7 @@
 
 ## Gradle Setup
 
-### project:build.gradle
+#### $PROJECT_ROOT/android/app/build.gradle
```

Comment:
I think you meant `$PROJECT_ROOT/android/build.gradle`?

Reply:
I'm not sure. I found 2 `build.gradle` files:

```
$ find . -name build.gradle
./android/app/build.gradle
./android/build.gradle
```

Which one should it be? I'll update the patch accordingly. :-)

Reply:
`./android/app/build.gradle` is the right one.

Reply:
Thanks, looks good! :)

Reply:
The first path should point to `$PROJECT_ROOT/android/build.gradle` the one below this one is correct by pointing to `$PROJECT_ROOT/android/app/build.gradle`